flags.go

```diff
@@ -74,6 +74,22 @@ func newApp() *cli.App {
 			Action:  listUpdatesCmd,
 		},
 		{
+			Name:    "update",
+			Aliases: []string{"up"},
+			Usage:   "Install the available updates",
+			Action:  updateCmd,
+			Flags: []cli.Flag{
+				cli.BoolFlag{
+					Name:  "l, auto-agree-with-licenses:",
```

Comment:
Maybe this can be merged with the global option `--non-interactive`. I would even change this global option to `-y` (yes to all), and then this new `-l` flag wouldn't be needed.